Archipelago.MultiClient.Net.Analyzers/Archipelago.MultiClient.Net.Analyzers/Analyzers/ItemFlagsComparisonInSwitchCaseDiagnostics.cs

```diff
@@ -4,25 +4,26 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Diagnostics;
 using System.Collections.Immutable;
+using System.Linq;
 
 namespace Archipelago.MultiClient.Net.Analyzers.Analyzers
 {
-    // TODO - how much of this applies to switch expressions or pattern matching switch statements?
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
-    public class ItemFlagsInSwitchDiagnostics : DiagnosticAnalyzer
+    public class ItemFlagsComparisonInSwitchCaseDiagnostics : DiagnosticAnalyzer
```

Comment:
Class name should match the file name. I also think this name is a bit clunky though the only alternatives coming to mind are worse

Reply:
It does match the file name, doesn't it? I don't think I'm missing something there. The name is indeed clunky, I'll see if I can come up with something better but I probably won't think about it too hard.

Reply:
For some reason I thought it didn't, I must have misread the diff